sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -209,6 +250,8 @@ namespace Azure { namespace Core { namespace Http {
      */
     virtual void OnUpgradedConnection(std::unique_ptr<CurlNetworkConnection>&&){};
 
+    void ValidateKeepAliveHeaders(Request& request, std::unique_ptr<RawResponse>& response);
```

Comment:
Why aren't `request` and `response` const references? Does `ValidateKeepAliveHeaders` modify the request and/or response? If so, can you explain what modifications are done in the comments?

Reply:
done